Documentation/platforms/aws.md

```diff
@@ -0,0 +1,77 @@
+running an instance, within the same Region, is “Unavailable” to you.
+
+In the context of an etcd cluster this means you must have at minimum a three member cluster where EC2 instances are spread across at least two AZs in order for an etcd cluster to be considered highly available at a Regional level.
+
+For most use cases the additional latency associated with a cluster spanning across Availability Zones will introduce a negligible performance impact.
+
+Availability considerations apply to all components of an application; if the application which accesses the etcd cluster will only be deployed to a single Availability Zone it may not make sense to make the etcd cluster highly available across zones.
+
+### Data Safety
+
+There are multiple mechanisms by which data safety can be achieved when deploying etcd on AWS.
+
+* replication: etcd replicates all data to all members of the etcd cluster therefore more nodes and disks you have in your cluster and the stronger the isolation between failure domains, the smaller the probability that data persisted to an etcd cluster will be unavailable to an application
+* Point in time etcd snapshotting: the etcd v3 API introduced support for snapshotting clusters. The operation is cheap enough (completing in the order of minutes) to run quite frequently and the resulting archives can be archived in a storage service like Amazon Simple Storage Service (S3).
+* Choosing Amazon Elastic Block Storage (EBS) rather than EC2 Instance Store: an EBS volumes is a replicated network attached block device. As a replicated storage offering, EBS volumes have stronger storage safety guarantees than InstanceStore or Ephemeral volumes which have a life cycle associated with the life cycle of the attached EC2 instance. The life cycle of an EBS volume is not necessarily tied to an EC2 instance and can be detached and snapshotted independently which means that a single node etcd cluster backed by an EBS volume can provide a fairly reasonable level of data safety.
+
+### Performance/Throughput
+
+The performance of an etcd cluster is roughly quantifiable through latency and throughput metrics which are primarily affected by disk and network performance. Detailed performance planning information is provided in the [performance section](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/performance.md) of the etcd operations guide.
+
+#### Network
+
+AWS offers EC2 Placement Groups which allow the collocation of EC2 instances within a single Availability Zone which can be utilized in order to minimize network latency between etcd members in the cluster. It is important to remember that collocation of etcd nodes within a single AZ will have consequences for the overall availability of the etcd cluster. [Enhanced networking for EC2 instances](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html) may also improve network performance of individual EC2 instances.
```

Comment:
s/will have consequences for the overall availability of the etcd cluster./will have weaker fault tolerance than distributing members across multiple AZs.